### PR TITLE
[beta-updates] android_splash_screen

### PR DESCRIPTION
**Pull request by craiglabenz**
Updates `android_splash_screen` for Dart 2.17

### CODE REVIEW

**Comment by craiglabenz** [comment]
Note that I didn't delete and recreate the `android` directory here because I thought it could possibly contain config necessary for the splash screen.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
There is something seriously wrong with CI. The  change in `platform_design/lib/songs_tab.dart` should fail on stable.

File: android_splash_screen/lib/main.dart [hunk 1]
```diff
@@ -42,10 +42,10 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  MyHomePageState createState() => MyHomePageState();
```
Comment (domesticmouse) [inline android_splash_screen/lib/main.dart:45]: Change to `State<MyHomePage> createState() => _MyHomePageState();`

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: android_splash_screen/lib/main.dart (hunk 1 above)
Comment (craiglabenz) [reply android_splash_screen/lib/main.dart]: Updated

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**